tests/test_nusantara.py

```diff
@@ -69,12 +69,12 @@
 }
 
 
-def _get_example_text(example: dict) -> str:
+def _get_example_text(example: dict, subcomponent: str = "passages") -> str:
```

Comment:
I don't think we need to change this one, we should actually keep it with `passages` such that the entities offset is based on the actual passages.